zebra/zebra_nb_config.c

```diff
@@ -3630,6 +3631,125 @@ int lib_vrf_zebra_ipv6_resolve_via_default_destroy(struct nb_cb_destroy_args *ar
 	return NB_OK;
 }
 
+/*
+ * XPath: /frr-vrf:lib/vrf/frr-zebra:zebra/netns/table-range
+ */
+static int table_range_validate(uint32_t start, uint32_t end, char *errmsg,
+				size_t errmsg_len)
+{
+#if defined(GNU_LINUX)
```

Comment:
It would be nice to be able to put these restrictions into the model, and get rid of the conditional compile. I wonder how important not having them is for the BSD folks...

Reply:
It would be a backward incompatible change if someone already has these tables configured on BSD. I really doubt that, but still, I don't want to do any changes like this as part of this PR.